js/dataframe/shared/VirtualTable.svelte

```diff
@@ -56,14 +56,18 @@
 		end = i;
 		const remaining = _items.length - end;
 
+		const scrollbarHeight = viewport.offsetHeight - viewport.clientHeight;
+		if (scrollbarHeight > 0) {
+			content_height += scrollbarHeight;
+		}
+
```

Comment:
The only issue with this is that a scrollbars are often dynamic, hiding and showing in response to interaction.

Reply:
From my tests, this didn’t appear to be an issue. Does this not work for you?

Reply:
I've only reviewed code so far, will test properly in the morning.

Reply:
sg thanks